modules/nf-core/wipertools/fastqgather/tests/main.nf.test

```diff
@@ -0,0 +1,72 @@
+nextflow_process {
+    name "Test Process WIPERTOOLS_FASTQGATHER"
+    script "../main.nf"
+    process "WIPERTOOLS_FASTQGATHER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "wipertools"
+    tag "wipertools/fastqgather"
+    tag "wipertools/fastqscatter"
+
+    setup {
+        run("WIPERTOOLS_FASTQSCATTER") {
+            script "../../fastqscatter/main.nf"
+            process {
+                """
+                input[0] = [
+                [ id:'pp_' ], // meta map
```

Comment:
Why does this id have an underscore at the end? That wouldn't be typical.

Reply:
if you read below, the test has an arg that accepts a prefix "pp_". The tool filters only file names beginning with that prefix. This is a test and we might tolerate this atypical prefix, I guess